src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -1146,6 +1146,59 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/request-content-url':
+    post:
+      x-alfresco-since: "7.0.0"
+      tags:
+        - nodes
+      summary: Generate a direct access content url
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.0 and newer versions.
+
+        Generate a direct access content url for the given **nodeId**.  Also, an optional expiry date could be set, so the direct access link would become invalid when the expiry date is reached. For example:
+
+        ```JSON
+         {
+           "expiresAt": "2017-03-23T23:00:00.000+0000"
+         }
+        ```
+
+        **Note:** If an expiry date isn't provided then a default of 28800 seconds (8 hours) link validity time is used if not configured otherwise.
+        **Note:** It is up to the actual ContentStore implementation if it can fulfil this request or not.
+      operationId: request-content-url
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - in: body
+          name: requestContentUrlBodyCreate
+          description: |
+            Optionally, specify the length of time in seconds that the link is valid for.
```

Comment:
We still have the old description in here, same for the other end points too.

Reply:
Updated now